packages/react/src/Tiptap.tsx

```diff
@@ -0,0 +1,109 @@
+import type { ReactNode } from 'react'
+import { createContext, useContext, useEffect, useState } from 'react'
+
+import type { Editor, EditorContentProps, EditorStateSnapshot } from './index.js'
+import { EditorContent, useEditorState } from './index.js'
+import { type BubbleMenuProps, BubbleMenu } from './menus/BubbleMenu.js'
+import { type FloatingMenuProps, FloatingMenu } from './menus/FloatingMenu.js'
+
+export type TiptapContextType = {
+  editor: Editor
+  isReady: boolean
```

Comment:
Could you add a documentation comment explaining what `isReady` means? Like, for example, "true when the editor has been created".

Also not sure if the `isReady` is necessary. It could be argued that, if the user wants to listen to when the editor has been created, they could just register an event listener. I think many developers will not use the `isReady` property.

Reply:
I will add JSDocs, I think I forgot them while working on it.

I would keep the isReady because in the future I'd like to use it for collaboration as well (so isReady would be true when the editor is initialized & the collaborative connection is ready if used).